src/server/Server.rei

```diff
@@ -83,4 +87,4 @@ module Infix: {
       Middleware.t('b, 'c)
     ) =>
     t([ | `With_middleware], [ | `Responded], 'a, 'c);
-};
+};
```

Comment:
Just a small change, but could you make sure all files include a newline at the end? 💯

Reply:
I couldn't find a way to configure that, I guess this is what refmt does by default?
Anyway, I configured vscode to always add newlines upon saving.